Assets/MRTK/Providers/Experimental/WindowsSceneUnderstanding/WindowsSceneUnderstandingObserver.cs

```diff
@@ -760,20 +785,41 @@ private async Task RunObserverAsync(CancellationToken cancellationToken)
         /// Gets the matrix representing the transform from scene space to world space
         /// </summary>
         /// <returns>The transform matrix</returns>
-        private System.Numerics.Matrix4x4 GetSceneToWorldTransform()
+        private System.Numerics.Matrix4x4? GetSceneToWorldTransform()
         {
             var result = System.Numerics.Matrix4x4.Identity;
 #if WINDOWS_UWP
```

Comment:
This is maybe a broader question, but do you know if the SU SDK should work over remoting, but it doesn't here because we `#if` it all out in the editor?

Reply:
I doubt that is the case here as if that were the case I should be seeing meshes appearing at random places during remoting, which was not the case when I tried.